genestack_client/settings/genestack_config.py

```diff
@@ -194,5 +172,62 @@ def save(self):
         with open(config_path, 'w') as f:
             document.writexml(f, indent='', addindent='    ', newl='\n')
 
+    def _store_value_insecurely(self, value, document, parent, element_name):
+        """
+        Store value in XML config file.
+
+        :param value: value to be stored
+        :type value: basestring
+        :param document: document
+        :type document: xml.dom.minidom.Document
+        :param parent: parent element
+        :type parent: xml.dom.minidom.Element
+        :param element_name: name of the element
+        :type element_name: basestring
+        :return: None
+        """
+        if self.store_raw is not None:
+            save_to_file = self.store_raw
+        elif self.store_raw_session is not None:
+            save_to_file = self.store_raw_session
+        else:
+            try:
+                save_to_file = ask_confirmation(
+                    'Do you want to store sensitive data in config file in plain text?',
+                    default='n')
+            except KeyboardInterrupt:
+                save_to_file = False
+            try:
+                self.store_raw = ask_confirmation('Set this as default behaviour?', default='y')
+            except KeyboardInterrupt:
+                self.store_raw_session = save_to_file
+        if save_to_file:
+            value_element = document.createElement(element_name)
+            value_element.appendChild(document.createTextNode(value))
+            parent.appendChild(value_element)
+        else:
+            sys.stderr.write('"%s" has not been saved to config file\n' % element_name)
+
+    def _store_value_securely(self, alias, secret_value, key):
```

Comment:
1. I think we must use the same argument order as `keyring.set_password()`.
2. These argument names are weird and unhelpful
3. Why do we have `key` here at all?  It's module constant, do we expect to save values into otherwise named *subvaults*, not only "Genestack SDK"?

So I propose:
```py
def _store_value_securely(self, key, value):
    """
    Save value in security vault under ``KEYRING_ENTRY``.
    """
    ...
        keyring.set_password(KEYRING_ENTRY, key, value)
```
Arg description in docstring have to be updated too, obviously.

Reply:
I want to save tokens and password with different keys.